src/Consolonia.Core/Controls/ConsoloniaAccessText.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using Avalonia.Media;
+using Avalonia.Controls.Primitives;
+using Avalonia.Controls.Documents;
+
+namespace Consolonia.Core.Controls
+{
+
+    /// <summary>
+    /// A text block that displays a character prefixed with an underscore as an access key.
+    /// </summary>
+    public sealed class ConsoloniaAccessText : AccessText
+    {
+        private Run _accessRun;
+
+        public ConsoloniaAccessText()
+        {
+            this.PropertyChanged += OnPropertyChanged;
+        }
+
+        private void OnPropertyChanged(object sender, Avalonia.AvaloniaPropertyChangedEventArgs e)
+        {
+            switch (e.Property.Name)
+            {
+                case nameof(Text):
+                    {
+                        if (!String.IsNullOrEmpty(Text))
+                        {
+                            _accessRun = null;
+                            InlineCollection inlines = new InlineCollection();
+                            var iPos = Text.IndexOf('_', StringComparison.Ordinal);
+                            if (iPos >= 0 && iPos < Text.Length - 1)  
+                            {
+                                inlines.Add(new Run(Text.Substring(0, iPos)));
```

Comment:
Just a hint: Text[..iPos] or Text[iPos..]

Reply:
oh, fancy. I totally forgot about that new feature. It hasn't been integrated into my coding patterns.

Reply:
Holy shit, I just tried it...so clean, so readable, so succinct. .Substring always felt so verbose. I love it!  Thanks for the tip!